server/index.js

```diff
@@ -278,14 +278,14 @@ app.get('/product/:id/related', async (req, res) => {
 
       response = await axios.get(`${APIurl}products/${relatedId}`, options)
       const product = response.data
+      const features = response.data.features
 
       response = await axios.get(`${APIurl}products/${relatedId}/styles`, options)
       const defaultStyle = response.data.results.find(style => style['default?']) || {}
```

Comment:
Server tweaking in order to grab features instead of characteristics for related products